plugins/com.google.cloud.tools.eclipse.sdk.ui/src/com/google/cloud/tools/eclipse/sdk/ui/InstallManagedCloudSdkStartup.java

```diff
@@ -0,0 +1,28 @@
+package com.google.cloud.tools.eclipse.sdk.ui;
```

Comment:
Why is this in the .ui subpackage? because it runs in the UI thread?

Reply:
It doesn't make sense for this to happen without the Eclipse Workbench (the UI).